web_shortcut/static/src/js/web_shortcut.js

```diff
@@ -6,6 +6,7 @@ odoo.define('web.shortcut', function (require) {
         WebClient = require('web.WebClient'),
         ViewManager = require('web.ViewManager'),
         ActionManager = require('web.ActionManager'),
+        Menu = require('web.Menu'),
```

Comment:
I reverted my changes to keep it compatible with Enterprise. My changes were to fix some bugs.